### DISCUSSION TITLE
Welcome to DeepPlastic Discussions!

### DISCUSSION THREAD

**Discussion by gautamtata** [gautamtata/DeepPlastic#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Deadpool5549** [comment]
Hi, Thanks for making the dataset public, I am using it to train my model on Kaggle , I have uploaded your dataset there, how can I add any citation or can you upload it on Kaggle so I can delete it.
